doc-content/shared-kie-docs/src/main/asciidoc/KieServer/Install-section.adoc

```diff
@@ -67,7 +67,12 @@ container. The Web container version of the WAR contains only the REST interface
 
 . Download and unzip the WildFly distribution. Let's call the root of the distribution ``WILDFLY_HOME``. This directory is named after the WildFly version, so for example ``wildfly-14.0.1.Final``.
 . Download _kie-server- -ee7.war_ and place it into ``WILDFLY_HOME/standalone/deployments``.
-. Configure user(s) and role(s). Execute the following command `` WILDFLY_HOME/bin/add-user.[sh|bat] -a -u 'kieserver' -p 'kieserver1!' -ro 'kie-server'``. You can of course choose different user name and password, just make sure that the user has role ``kie-server``.
+. Configure user(s) and role(s) by executing the following command:
```

Comment:
Never use (s) to indicate plural. I would just say configure users and roles....